### ISSUE TITLE
nas

### ISSUE THREAD

**Issue by nassimcnn** [shivamsaboo17/Overcoming-Catastrophic-forgetting-in-Neural-Networks#4]
what is crit argument in ewc = ElasticWeightConsolidation(model, crit, lr=0.01, weight=0.1)? and how to build it ?

**Comment by ThomasAtlantis** [comment]
It's short for `criterion`, i.e. the basic loss function. As shown by `demo.ipynb`, you can build a cross entropy loss for classification tasks: `crit = nn.CrossEntropyLoss()`.
